### DISCUSSION TITLE
Compatibility with Multi-Vendor plugins

### DISCUSSION THREAD

**Discussion by Mrlunsuper** [woocommerce/woocommerce#34919] [WooCommerce HPOS]
Hi there. I'm a newbie. I would like to ask how will the new HPOS system break Multi-Vendor plugins since most of those plugins rely on author_id to identify vendor orders and you don't use the wp_posts table anymore.
Best regards!

**Comment by vedanshujain** [comment]
Hey @Mrlunsuper , we currently don't migrate the post_author field since the WooCommerce does not use it, although we can consider it. Meanwhile, since we store a placeholder post for every order, you can continue the use the field in the posts table. 

Can you please open an issue to consider migrating post_author field, we will continue our discussion there.
